dids/src/main/kotlin/web5/sdk/dids/exceptions/PkarrRecordResponseException.kt

```diff
@@ -6,3 +6,10 @@ package web5.sdk.dids.exceptions
  * @param message the exception message detailing the error
  */
 public class PkarrRecordResponseException(message: String) : RuntimeException(message)
+
+/**
+ * Did resolution exception.
+ *
+ * @param message the exception message detailing the error
+ */
+public class DidResolutionException(message: String) : RuntimeException(message)
```

Comment:
Should this be in its own exception file?

Reply:
Done in [cb911e9](https://github.com/TBD54566975/web5-kt/pull/172/commits/cb911e9a37ca2de4f8e063e6bb1cab7e084d8fb9)